### ISSUE TITLE
0.4.1 does no longer compile on latest nightly

### ISSUE THREAD

**Issue by SirVer** [serde-rs/aster#25]
This is on Mac OS X.

```
$ cargo build                                                                                      
    Updating registry `https://github.com/rust-lang/crates.io-index`
   Compiling aster v0.4.1 (file:///private/tmp/rust-aster)
src/mac.rs:106:18: 106:48 error: this function takes 4 parameters but 3 parameters were supplied [E0061]
src/mac.rs:106     let mut cx = ExtCtxt::new(&sess, cfg, ecfg);
                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/mac.rs:106:18: 106:48 help: run `rustc --explain E0061` to see a detailed explanation
error: aborting due to previous error
Could not compile `aster`.
```

```
$ rustc --version                                                                                                            
rustc 1.4.0-nightly (e35fd7481 2015-08-17)
```

**Comment by erickt** [comment]
Fixed in 0.4.2!

**Comment by SirVer** [comment]
Thanks!
